tracing-attributes/src/lib.rs

```diff
@@ -97,8 +97,11 @@ use syn::{
 ///
 /// Unless overriden, a span with `info` level will be generated.
 /// The generated span's name will be the name of the function. Any arguments
-/// to that function will be recorded as fields using [`fmt::Debug`]. To skip
-/// recording a function's or method's argument, pass the argument's name
+/// to that function will be recorded as fields using [`fmt::Debug`] except to
+/// primitive types including `bool`, `u8`, `i8`, `u16`, `i16`, `u32`, `i32`,
+/// `u64`, `i64`, `usize`, and `isize`.
+///
+/// To skip recording a function's or method's argument, pass the argument's name
 /// to the `skip` argument on the `#[instrument]` macro. For example,
 /// `skip` can be used when an argument to an instrumented function does
 /// not implement [`fmt::Debug`], or to exclude an argument with a verbose
```

Comment:
Maybe this should be changed to something like "... which does not implement [`fmt::Debug`] or [`tracing::field::Value`], or ..."?